src/app/shared/modules/wrapper.module.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ *    Copyright 2017 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { MarkdownModule } from 'ngx-markdown';
+import { MarkdownWrapperComponent } from '../markdown-wrapper/markdown-wrapper.component';
+
+@NgModule({
+  declarations: [MarkdownWrapperComponent],
+  imports: [CommonModule, MarkdownModule],
+  providers: [],
+  exports: [MarkdownWrapperComponent],
+  entryComponents: [],
+})
+export class WrapperModule {}
```

Comment:
Suggest the name be less generic, like MarkdownWrapperModule. Also update the filename. One should be able to have slightly more specific sense of what it does from the name.

Reply:
Updated to MarkdownWrapperModule